pkg/webservice/routes.go

```diff
@@ -126,6 +126,14 @@ var webRoutes = routes{
 		getContainerHistory,
 	},
 
+	//
+	route{
+		"Partitions",
+		"GET",
+		"/ws/v1/partitions",
+		getPartitions,
+	},
```

Comment:
Are we going to support the sub-path `ws/v1/partitions/{partitionName}`?

Reply:
As of now, We don't have such plan.